src/widgets/stats/defaultTemplates.ts

```diff
@@ -0,0 +1 @@
+export { defaultTemplates as default } from './stats';
```

Comment:
what about importing it in the widget file instead?

Reply:
Do you mean the other way around? We do it this way in the latest `refinementList` update, but the other way in other files.

Which one should be the one we stick to?

Reply:
good point, it's the same for me actually, in refinement list I moved them inside since I did some more logic, but not sure what's best